### PR TITLE
added toggle-able desktop notification option - closes #4

### PR DESCRIPTION
**Pull request by albertofp**
Not sure if theres anything that could be done better/differently first time contributing.

### CODE REVIEW

File: src/main/java/com/WildernessPlayerAlarm/WildernessPlayerAlarmConfig.java [hunk 1]
```diff
@@ -55,4 +55,17 @@ default Color flashColor()
 	{
 		return new Color(255, 255, 0, 70);
 	}
+
+	@ConfigItem(
```
Comment (adhansen) [inline src/main/java/com/WildernessPlayerAlarm/WildernessPlayerAlarmConfig.java]: Would you change this to `position = 1`, move it up in the config, and increment the position of the settings that are then below?

File: src/main/java/com/WildernessPlayerAlarm/WildernessPlayerAlarmPlugin.java [hunk 2]
```diff
@@ -63,6 +67,9 @@ public void onClientTick(ClientTick clientTick) {
 
 		if (shouldAlarm && !overlayOn)
 		{
+			if (config.toggleNotification()){
+				notifier.notify("Player Spotted!");
```
Comment (adhansen) [inline src/main/java/com/WildernessPlayerAlarm/WildernessPlayerAlarmPlugin.java]: Super nit: can the casing on `Spotted` be changed to `spotted`?

File: src/main/java/com/WildernessPlayerAlarm/WildernessPlayerAlarmConfig.java [hunk 3]
```diff
@@ -55,4 +55,17 @@ default Color flashColor()
...
+			keyName = "toggleNotification",
+			name = "Desktop Notification",
```
Comment (adhansen) [inline src/main/java/com/WildernessPlayerAlarm/WildernessPlayerAlarmConfig.java]: "toggleNotification" -> "desktopNotification"
"Desktop Notification" -> "Desktop notification"

**Review by adhansen — CHANGES_REQUESTED** [review]

**Comment by albertofp** [comment]
All done :)

**Review by adhansen — APPROVED** [review]

**Pull request merged**